src/app/src/Map.js

```diff
@@ -6,13 +6,25 @@ import ReactMapGL, {
 } from 'react-map-gl';
 import 'mapbox-gl/dist/mapbox-gl.css';
 import { connect } from 'react-redux';
+import mapboxgl from 'mapbox-gl';
 
 import { updateViewport } from './map.actions';
 import SensorMarker from './SensorMarker';
 import BackToMapButton from './BackToMapButton';
 import { toggleBackToMapButton } from './map.actions';
 
 class GLMap extends Component {
+    constructor() {
+        super();
+
+        this._mapRef = React.createRef();
```

Comment:
What does the `_` signify? I think I had read it was an informal signifier for a private var.

Reply:
Yeah, that's correct. I go back and forth on using it. I don't think it's adding much value here, so I'm going to remove it. Thanks.